backend/src/main/java/hu/flowacademy/vajdasagbrand/service/KeycloakClientService.java

```diff
@@ -51,10 +76,20 @@ public AccessTokenResponse login(String email, String password) {
                 .getAccessToken();
     }
 
+    public boolean enableUser(String username) {
+        UsersResource resource = keycloak.realm(keycloakPropertiesHolder.getKeycloakBackendClientRealm()).users();
+        List<UserRepresentation> listOfUsers = resource.search(username);
+        if (listOfUsers.isEmpty()) return false;
+        UserRepresentation userToEnable = listOfUsers.get(0);
+        userToEnable.setEnabled(true);
+        resource.get(userToEnable.getId()).update(userToEnable);
+        return true;
+    }
+
     public boolean deleteUser(String username) {
         UsersResource resource = keycloak.realm(keycloakPropertiesHolder.getKeycloakBackendClientRealm()).users();
         List<UserRepresentation> listOfUsers = resource.search(username);
-        if(listOfUsers.isEmpty()) return false;
+        if (listOfUsers.isEmpty()) return false;
```

Comment:
Ne legyen inline if, rakd ki a zárójeleket kérlek. Itt is használható lenne a `stream().findFirst().map(...)`

Reply:
kijavítva